### DISCUSSION TITLE
I'm not a Ruby programmer

### DISCUSSION THREAD

**Discussion by d351r3d** [postalserver/postal#1677] [Help with using Postal]
I'd possibly like to use Postal as it seems fit for the thing I want it to do. I want it to be a replacement for protonmail and have more than good security. I would prefer it to only send some emails from Ghost and Flarum in SMTP and be a noreply (Postal will send and not receive). I also need say 15 emails that will only be able to send and receive text, I don't want any attachments being sent or received.

Also I was wondering what's with the 100gb requirement for installation? Could you set the whatever is saved to be deleted after being sent or received after a month? I think that's enough time to see am email and respond. Emails that go out just go out. No emails will be coming in for automated website stuff.

**Comment by willpower232** [comment]
It sounds like Postal will be useful to you, enabling website CMSes to send emails without worrying too much about technicalities.

Regarding disk space usage, you can probably survive with less but it is easy to run out of disk space and 100GB adds a nice safety margin. Have a look at #1627

**Reply by d351r3d** [reply]
I read #1627 and it was good but I still have a question. If the disk space is mostly database then could you prune it say every month after something is sent. Say a lifetime of any email is a month and then it's deleted. I don't need to keep email forever and if and when I do I'll figure out how to save it neatly.

https://ghost.org/docs/faq/mailgun-newsletters/ I would like to know if Postal has Railgun (bulk mail) functionality since that would be nice because Ghost does use it.

Also is there a demo because I want to see it in action and let my people see it so they can see for themselves. You can see a video online but when you get to click stuff it's different. If you don't it's fine, I understand. I hear Postal has a fancy admin and I like that a lot.

**Reply by willpower232** [reply]
Postal can already regulate email storage to keep database usage down but you'll need to keep more of an eye on it if you are on a smaller server.

![image](https://user-images.githubusercontent.com/1619102/139646244-a0cdcd1f-7b86-40c7-a170-bace8f12d3b3.png)

Postal could be used to send bulk email newsletters however you would need to manage unsubscribes and other contact list functionality yourself.

There is not a demo website but with the latest version being powered by docker, hopefully it is easy to set up a small server yourself.

**Reply by d351r3d** [reply]
Right on. Thanks for the assistance.
